jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-showcase/README.md

```diff
@@ -79,4 +79,30 @@ Start the server by running:
 
 Verify the application is running. Go to http://SERVER:PORT/jbpm-cm/ and type the specified username and password. You should see simple XML message with basic information about the server.
 
-For more details, please check the [Getting Started Guide](https://docs.jboss.org/author/display/WFLY10/Getting+Started+Guide).
\ No newline at end of file
+For more details, please check the [Getting Started Guide](https://docs.jboss.org/author/display/WFLY10/Getting+Started+Guide).
+
+## Demo Docker image
+
+To allow you to quickly get started with the jBPM Case Management Showcase fully integrated with a running Kie Server, a Docker image can be generated as part of the build.
+This image contains both applications deployed to a single WildFly 10 instance. It also provides the settings necessary for showcase to retrieve the necessary data from the running Kie Server instance.
+By default, a few demo containers are deployed to allow you to interact with the Case Management concepts as well as a default user with the following credentials: admin/admin. 
```

Comment:
Could we add one more user to be able to also check some scenarios requiring more users? For example case comments should be displayed differently for user who didn't add them etc.